packages/test-support/src/project.ts

```diff
@@ -174,6 +174,31 @@ export function emberAppTemplate(): Project {
   });
 }
 
+export function getEmber4AppProject(): Project {
+  const project = getEmberAppProject();
+
+  const importRoot = dirname(fileURLToPath(import.meta.url));
+  const result = importRoot.match(/.+test-support/);
+  if (!result || !Array.isArray(result)) {
+    return project;
+  }
+  const testSupport = result[0];
+
+  project.linkDevDependency('ember-source', {
+    baseDir: testSupport,
+    resolveName: 'ember-source-4.4',
```

Comment:
if you change above these will have a different resolve name to match

Reply:
Changed